tests/test_core.py

```diff
@@ -139,6 +139,18 @@ def can_do_standalone(self):
       '-fsanitize=address' not in self.emcc_args
 
 
+def also_with_wasmfs(func):
+  def decorated(self):
+    self.set_setting('WASMFS', 0)
```

Comment:
I don't think you need this line.

Reply:
I was putting it there for clarity, but I guess it's redundant